plugins/MauticFocusBundle/EventListener/PageSubscriber.php

```diff
@@ -41,28 +40,28 @@ class PageSubscriber implements EventSubscriberInterface
     private $security;
 
     /**
-     * Must be there until BuilderTokenHelper is refactored.
-     *
-     * @var MauticFactory
+     * @var BuilderTokenHelperFactory
      */
-    private $factory;
+    private $builderTokenHelperFactory;
 
     /**
-     * @param FocusModel      $model
-     * @param RouterInterface $router
-     * @param CorePermissions $security
-     * @param MauticFactory   $factory
+     * PageSubscriber constructor.
```

Comment:
I'm deleting these comments whenever I see them. It's duplicated information that doesn't bring any value. I know that some IDEs import it automatically, but can you configure yours to stop do that?

Reply:
The same could be said about the docblocks that don't add anything given the method signatures define the types as well - should we remove those? (I'm already in the habit of removing them...)

Reply:
Good point. With PHP 7.2 there is no need for them as we can define even scalar types in the method signature. 👍